lib/ci/test_reporters/tap_reporter.js

```diff
@@ -19,17 +20,24 @@ TapReporter.prototype = {
     })
     this.display(prefix, data)
     this.total++
-    if (data.passed) {
+    if (data.skipped) {
+      this.skipped++
+    }
+    else if (data.passed) {
       this.pass++
     }
   },
   summaryDisplay: function(){
     var lines = [
-      '1..' + this.total,
-      '# tests ' + this.total,
-      '# pass  ' + this.pass,
-      '# fail  ' + (this.total - this.pass)
+      '1..' + this.total
     ]
+    lines.push('# tests ' + this.total)
+    lines.push('# pass  ' + this.pass)
+    if (this.skipped > 0) {
+      lines.push('# skip  ' + this.skipped)
```

Comment:
I would recommend the number of skipped tests always be displayed, even if it is 0.